README.md

```diff
@@ -36,7 +36,16 @@ dart pub add flutter_stripe
 
 #### Android
 
-Android 5.0 (API level 21) and above
+- Android 5.0 (API level 21) and above
```

Comment:
also Kotlin 1.5.0 or later need to be used because there's seemingly an incompatibility with the Stripe Android library otherwise